qemu/tests/cfg/live_snapshot.cfg

```diff
@@ -42,3 +42,28 @@
             dd_cmd = "dd if=/dev/zero of=%s bs=4K count=1024 oflag=dsync"
```

Comment:
This is an old case which has already been in the acceptance loop.
And checked latest acceptance test results, test pass for windows guest.
15:07:05 INFO | PASS 076-smp_4.4096m.repeat1.Host_RHEL.m6.u8.spice.qcow2.virtio_scsi.up.virtio_net.Win10.x86_64.io-github-autotest-qemu.live_snapshot.base